### ISSUE TITLE
Request GitHub KYC-tgdaosheng

### ISSUE THREAD

**Issue by TgDaosheng** [DigitalPlatDev/FreeDomain#38058]
### KYC Using Your GitHub Account

Please make sure your GitHub account meets the following requirements (all are required):

- [x] The issue title must be: Request GitHub KYC - Your US.KG Panel registered username. This ensures our automated script can detect your registered username. (For example, your title should be:  **Request GitHub KYC-example**)
- [x] This is not a new GitHub account; it has existing repositories and stars.
- [x] This is my first US.KG NIC Panel account.
- [x] I promise not to use the domain name for criminal or abusive purposes. 
- [x] Please star this repository, This will let more people know about this project!

**Your Account Details (US.KG NIC Panel):**

Please fill out the following information; otherwise, your request will not be processed:

- Username: tgdaosheng
- Reason for Registering the .US.KG Domain Name (Why):I am a student learning programming skills and applying for a domain name to build a project website.

**Comment by TgDaosheng** [comment]
Hello! I hope I haven't disturbed your busy work. I checked that my application has not been approved yet, and I am particularly anxious because many subsequent plans are closely related to it.
I carefully reviewed the application process and submitted materials, and still feel confused about where the problem lies. I assure you that I have maintained a sincere and compliant attitude throughout the entire application process, and strictly followed all requirements. Thank you very much for your trouble. I look forward to your feedback and wish you a happy life and smooth work!
